style: apply /style-guide pass to weave (#2739)
wandb/docs: weave/guides/tracking/trace-sub-agents.mdx

---
title: Trace sub-agents

> description: Trace nested agents and agents delegated tasks using the Weave sub-agent span.

Reviewer: ```suggestion
description: Use Weave's sub-agent span to trace sub-agent delegations and view nested agent invocations.
```

Author: Applied verbatim in f376e6074.
